frontend/core-ui/src/modules/notification/my-inbox/components/contents/system/WelcomeMessage.tsx

```
@@ -1,130 +1,272 @@
+          if (item.forOwner && !isOwner) {
+            return null;
+          }
+          return (
+            <div
+              className={cn(
+                'border rounded-2xl p-3 flex flex-col gap-3 bg-background flex-1 min-w-0',
+                index === 2 &&
+                  OnboardingSteps.length === 3 &&
+                  'lg:col-span-2 col-span-2 sm:col-span-1',
+              )}
+            >
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)